src/System.Drawing.Common/tests/Drawing2D/GraphicsPathTest.cs

```diff
@@ -0,0 +1,3294 @@
+﻿//
```

Comment:
@KostaVlev thanks for the contribution. Please prefix with these two lines:

// Licensed to the .NET Foundation under one or more agreements.
// See the LICENSE file in the project root for more information.